pkgs/games/quakespasm/default.nix

```diff
@@ -20,6 +20,12 @@ stdenv.mkDerivation rec {
     ./quakespasm-darwin-makefile-improvements.patch
   ];
 
+  # Quakespasm tries to set a 10.6 deployment target, but that’s too low for SDL2.
+  postPatch = lib.optionalString stdenv.hostPlatform.isDarwin ''
+    substituteInPlace Makefile.darwin \
+      --replace-fail '-mmacosx-version-min=10.6' "-mmacosx-version-min=$MACOSX_DEPLOYMENT_TARGET"
```

Comment:
We could simply remove this to avoid the dependency on the deployment target variable, right? And in that case it could be unconditional, as it only affects Darwin compilation anyway and keeping patches unconditional is good hygiene.

Reply:
Probably the way to go is to use sed to delete the lines that try to set the version. It uses different ones depending on the platform.

Reply:
Sounds reasonable. There’s already a patch to the Darwin Makefile so it could just be rolled into that.